test/spec/modules/eids_spec.js

```diff
@@ -342,6 +342,23 @@ describe('eids array generation for known sub-modules', function() {
       }]
     });
   });
+  it('originFloc', function() {
+    const userId = {
+      flocId: {id: 'Sample_Cohort', version: 'chrome1.1'}
+    };
+    const newEids = createEidsArray(userId);
+    expect(newEids.length).to.equal(1);
+    expect(newEids[0]).to.deep.equal({
+      source: 'floc.com',
```

Comment:
chrome.com and web.dev have been proposed in #6510 
google.com seems like a candidate as well. 
does anyone have a preference?

Reply:
i would vote for `chrome.com` as it's the browser providing this, and probably shouldn't be as broad as google.com. chrome.com vs web.dev, though, I don't have a strong opinion on

Reply:
I'll second `chrome.com`.